apps/src/templates/sectionProgress/VirtualizedDetailView.jsx

```diff
@@ -124,8 +141,14 @@ export default class VirtualizedDetailView extends Component {
     const tableHeightFromRowCount = rowHeight * rowCount;
     // Use a 'maxHeight' of 680 for when there are many rows
     const tableHeight = Math.min(tableHeightFromRowCount, 680);
+    const lessonNumbers =  Array(scriptData.stages.length).fill().map((e,i)=>i+1);
 
     return (
+      <div>
+        <LessonSelector
```

Comment:
Since this gets used in both summary and detail view, I think it makes more sense to add it to `SectionProgress` and manage the state there. Then each of the summary and detail view can get passed props `lessonOfInterest` and handle their own styling.